tests/integration/deploy/test_deploy_command.py

```diff
@@ -1198,6 +1209,33 @@ def test_deploy_logs_warning_with_cdk_project(self):
         self.assertIn(warning_message, deploy_process_execute.stdout)
         self.assertEqual(deploy_process_execute.process.returncode, 0)
 
+    def _assert_companion_stack(self, cfn_client, companion_stack_name):
+        try:
+            cfn_client.describe_stacks(StackName=companion_stack_name)
+        except ClientError:
+            self.fail("No companion stack found.")
+
+    def _assert_companion_stack_content(self, ecr_client, companion_stack_name):
+        stack = boto3.resource("cloudformation").Stack(companion_stack_name)
+        resources = stack.resource_summaries.all()
+        for resource in resources:
+            if resource.resource_type == "AWS::ECR::Repository":
+                policy = ecr_client.get_repository_policy(repositoryName=resource.physical_resource_id)
+                self._assert_ecr_lambda_policy(policy)
+            else:
+                self.fail("Non ECR Repo resource found in companion stack")
+
+    def _assert_ecr_lambda_policy(self, policy):
+        policy = json.loads(policy.get("policyText"))
```

Comment:
Let's add default value policy.get("policyText", {}) so that json.loads(None) never happens!

Reply:
Updated.